app/models/host/managed.rb

```diff
@@ -294,7 +294,7 @@ def handle_ca
   def import_facts(facts, source_proxy = nil)
     # Facts come from 'existing' attributes/infrastructure. We skip triggering
     # the orchestration of this infrastructure when we create a host this way.
-    skip_orchestration! if SETTINGS[:unattended]
+    skip_orchestration! if SETTINGS[:unattended] and not SETTINGS[:unattended_orchestration]
```

Comment:
Prefer `&&` over `and` (there is slight difference in operator preferrence). Also `!` is generally preferred in Foreman codebase.